src/bliss/bench_ART.h

```diff
@@ -28,17 +29,20 @@ class BlissARTIndex : public BlissIndex<KEY_TYPE, VALUE_TYPE> {
     }
 
     bool get(KEY_TYPE key) override { 
-        uint8_t ARTkey[8];
+        uint8_t ARTkey[KEY_SIZE];
         ART::loadKey(key, ARTkey);
 
-        ART::Node* leaf = ART::lookup(_index, ARTkey, 8, 0, 8);
+        uint8_t depth = 0;
+        ART::Node* leaf = ART::lookup(_index, ARTkey, KEY_SIZE, depth, KEY_SIZE);
         return ART::isLeaf(leaf) && ART::getLeafValue(leaf) == key;
      }
 
     void put(KEY_TYPE key, VALUE_TYPE value) override {
-        uint8_t ARTkey[8];
+        uint8_t ARTkey[KEY_SIZE];
         ART::loadKey(key, ARTkey);
-        ART::insert(_index, &_index, ARTkey, 0, key, 8);
+
+        uint8_t depth = 0;
+        ART::insert(_index, &_index, ARTkey, depth, key, KEY_SIZE);
```

Comment:
we are taking in value as a parameter and only adding <key, key> which can be confusing. Better change this to insert <key, value>. Inserting the k-v pair as <key, key> is a decision we make and not universal!